artscraper/artist_works_metadata.py

```diff
@@ -95,3 +97,133 @@ def get_artist_description(artist_link):
     description = page.summary
 
     return description
+
+def get_artist_metadata(artist_link):
+
+    '''
+    Parameters
+    ----------
+    artist_link : Web address of the artist's Google Arts & Culture page
+
+    Returns
+    -------
+    metadata: Dictionary containing metadata about the artist
+    '''
+
+    # Get Wikidata ID of artist
+    artist_id = get_artist_wikidata_id(artist_link)
+
+    # Wikidata database to query
+    url = 'https://query.wikidata.org/sparql'
+
+    # SPARQL query
+    with open('../docs/sparql_query.txt', 'r', encoding='utf-8') as file:
+        query = file.readlines()
```

Comment:
This is not needed anywhere else as a file, right? Then we should probably define it as a global string. Otherwise, you can only run the code in specific directories.

Another solution is to add it to the package, but that's a little bit more involved with setup.py/MANIFEST.in.

Reply:
I have now added the SPARQL query as a global string in the python file.